androidshared/src/main/res/layout/app_bar_layout.xml

```diff
@@ -1,4 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
+     When scrolling, the top app bar fills with a contrasting color to create a visual separation.
+     This works automatically if your scrolling view (e.g., `RecyclerView`, `ListView`) is placed directly
+     beneath the `AppBarLayout`. However, if the scrolling view is nested within another view
```

Comment:
Is it "beneath" or is it if the two are children in the same `CoordinatorLayout`? It might be worth also linking to the docs here: https://developer.android.com/reference/com/google/android/material/appbar/AppBarLayout.

Reply:
> Is it "beneath" or is it if the two are children in the same CoordinatorLayout?

Both :smile: They are children in the same `CoordinatorLayout` and that layout works in the way that displayed the app bar above and the scrolling view beneath. If the scrolling view is there without any wrapper everything is fine out of the box.

> It might be worth also linking to the docs here: https://developer.android.com/reference/com/google/android/material/appbar/AppBarLayout

This document does not emphasize the importance of the aspect mentioned in the comment at all, so I believe it is useless.